--- ./Makefile.org	Thu May  9 01:04:51 2002
+++ ./Makefile	Thu May  9 20:53:10 2002
@@ -4,7 +4,7 @@
 
 #CC = gcc
 
-DESTDIR=/usr/local/bin/
+BINDIR=/usr/X11R6/bin/
 BIN=3dfx_tvtool
 
 all:
@@ -14,8 +14,8 @@
 	rm -rf *~ src/*~ src/*.o bin/$(BIN)
 
 install:
-	install -d $(DESTDIR)
-	install -c -m 0755 bin/$(BIN) $(DESTDIR)
+	install -d $(DESTDIR)$(BINDIR)
+	install -c -m 0755 bin/$(BIN) $(DESTDIR)$(BINDIR)
 
 uninstall:
-	rm -f $(DESTDIR)/$(BIN)
+	rm -f $(DESTDIR)$(BINDIR)/$(BIN)
